packages/backend/src/server/web/views/base.pug

```diff
@@ -34,7 +34,8 @@ html
 		link(rel='prefetch' href='https://xn--931a.moe/assets/info.jpg')
 		link(rel='prefetch' href='https://xn--931a.moe/assets/not-found.jpg')
 		link(rel='prefetch' href='https://xn--931a.moe/assets/error.jpg')
-		link(rel='stylesheet' href='/assets/fontawesome/css/all.css')
+		//- link(rel='stylesheet' href='/assets/fontawesome/css/all.css')
```

Comment:
@syuilo Do you think you could provide some insight as t *where* this file (`/assets/fontawesome/css/all.css`) is? I can't seem to find it in the codebase.

Reply:
https://github.com/misskey-dev/misskey/blob/8df2f19b5b371d73809d12b2ffbcba3ec3cffb2d/packages/backend/src/server/web/index.ts#L113
↓

https://github.com/misskey-dev/misskey/blob/8df2f19b5b371d73809d12b2ffbcba3ec3cffb2d/packages/backend/src/server/web/index.ts#L35
↓

https://github.com/misskey-dev/misskey/blob/8df2f19b5b371d73809d12b2ffbcba3ec3cffb2d/gulpfile.js#L23